Wayofbeing models and testing 2

Zmuse001 opened the pull request:
rebased and seeded and tested

Zmuse001 commented:
I rebased the branch and merged it to the main and then i ran a rails db:seed and seeded my database with the dummy information and tested my models and ran rubocop to fix my errors so that everything is in order. 
https://app.asana.com/0/1206510424537042/1206833138919405/f

how do i link to issues? i don't think i know what that one means. i did everything else though. i believe

camckin10 commented:
> I rebased the branch and merged it to the main and then i ran a rails db:seed and seeded my database with the dummy information and tested my models and ran rubocop to fix my errors so that everything is in order. https://app.asana.com/0/1206510424537042/1206833138919405/f
> 
> how do i link to issues? i don't think i know what that one means. i did everything else though. i believe

@Zmuse001  You would just be linking your Asana ticket, not issues. 
Also, please check your DM for the content you need to add to the PR description. Thank you!

Zmuse001 commented:
Changes
created wayofbeing and studentwayofbeing models and tested them and used factory bot to create dummy data and seeded the data

Link to Issue
https://app.asana.com/0/1206510424537042/1206833138919405/f

Comments, Notes, or Questions
I still need more practice factory bot and testing. i was very uncomfortable with this ticket 

Related Screenshots
For FE tickets specifcially, please include a screenshot of your work!

Before you submit for review, Did you...
 Added relevant tests (if applicable)
 Linked Asana ticket to PR
 Added comments, notes, or questions for reviewwer
 Added relevant screenshots
 Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

camckin10 commented:
> Changes created wayofbeing and studentwayofbeing models and tested them and used factory bot to create dummy data and seeded the data
> 
> Link to Issue https://app.asana.com/0/1206510424537042/1206833138919405/f
> 
> Comments, Notes, or Questions I still need more practice factory bot and testing. i was very uncomfortable with this ticket
> 
> Related Screenshots For FE tickets specifcially, please include a screenshot of your work!
> 
> Before you submit for review, Did you... Added relevant tests (if applicable) Linked Asana ticket to PR Added comments, notes, or questions for reviewwer Added relevant screenshots Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

@Zmuse001  You need to place all of this content _inside_  of the content box where you posted 'rebased, seeded,etc'

edithemmings commented on app/models/way_of_being.rb:
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class WayOfBeing < ApplicationRecord
+  has_many :student_way_of_being, dependent: :destroy
```
```suggestion
  has_many :student_ways_of_being, dependent: :destroy
```
has_many should be plural

edithemmings reviewed (commented):
Looks great! Just one minor pluralization to make before merging

edithemmings reviewed (approved):
🎉🚀
